src/main/resources/reference.conf

```diff
@@ -156,4 +171,53 @@ cassandra-snapshot-store {
       parallelism-max = 8
     }
   }
+}
+
+# Configuration for the CassandraReadJournal
+cassandra-query-journal {
+  # Implementation class of the Cassandra ReadJournalProvider
+  class = "akka.persistence.cassandra.query.CassandraReadJournalProvider"
+  
+  # Absolute path to the write journal plugin configuration section
+  write-plugin = "cassandra-journal"
+  
+  # New events are retrieved (polled) with this interval.
+  refresh-interval = 3s
+  
+  # How many events to fetch in one query (replay) and keep buffered until they
+  # are delivered downstreams.
+  max-buffer-size = 100
+  
+  # The fetch size of the Cassandra select statement
+  fetch-size = 50
+  
+  # Read consistency level
+  read-consistency = "QUORUM"
+  
+  # Configure this to the day (yyyyMMdd) when the system was first started.
+  # When offset 0L is used it will look for events from this day and forward.
+  first-time-bucket = "20151120"
+
+  # The returned event stream is ordered by the offset (timestamp), which corresponds
+  # to the same order as the write journal stored the events, with inaccuracy due to clock skew
+  # between different nodes. The same stream elements (in same order) are returned for multiple
+  # executions of the query on a best effort basis. The query is using a Cassandra Materialized
+  # View for the query and that is eventually consistent, so different queries may see different
+  # events for the latest events, but eventually the result will be ordered by timestamp
+  # (Cassandra timeuuid column). To compensate for the the eventual consistency the query is
+  # delayed to not read the latest events, the duration of this delay is defined by this 
+  # configuration property.
+  # However, this is only best effort and in case of network partitions
+  # or other things that may delay the updates of the Materialized View the events may be
+  # delivered in different order (not strictly by their timestamp).  
+  eventual-consistency-delay = 10s
```

Comment:
I am not very in favour of this solution, but it seems sensible given the restrictions of Cassandra's MV. If set to 0 I assume the data would be delivered immediatelly with the possiblity of different results on multiple executions?